### PR TITLE
Updated Timeline datasets and examples to maintain consistency.

### LINKED ISSUE #214: Update Timeline graph example datasets and reflow logic to maintain consistency
# Story
JIRA: [UXPLATFORM-5468](https://jira2.cerner.com/browse/UXPLATFORM-5468)
## Description

As a developer, I would like to make changes to the example datasets and example reflow logic for timeline graphs.

## Motivation

So that consistency can be maintained across all the examples present in the terra-dev site.

## Definition of Done

We'll know we're done when the datasets and reflow logic for timeline examples are consistent:

- Reflow logic for both react and carbon-centric examples are updated to be consistent with this logic: https://github.com/cerner/terra-graphs/pull/193#discussion_r679991504 
- All datasets are updated to be non arrays	
- Dedicated graph config objects are added for the examples where they are dynamically updated

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Updated Timeline graph config objects, and datasets in examples to maintain consistency.

Closes #214 

JIRA LINK: https://jira2.cerner.com/browse/UXPLATFORM-5468

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

**Review by ryanthemanuel — APPROVED** [review]

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset1.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadTimelinePopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const bobRossIpsum = "Just use the old one inch brush. It's so important to do something every day that will make you happy. "
+  + "I sincerely wish for you every possible joy life could bring. Water's like me. It's laaazy ... Boy, it "
+  + "always looks for the easiest way to do things\n\nWe're trying to teach you a technique here and how to use it. "
+  + 'The least little bit can do so much. There we go.\n\nNo worries. No cares. Just float and wait for the wind to blow you around. '
+  + "These trees are so much fun. I get started on them and I have a hard time stopping. We'll make some happy little bushes here. "
+  + 'Absolutely no pressure. You are just a whisper floating across a mountain. There are no limits in this world. '
+  + 'A thin paint will stick to a thick paint.\n';
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset1.js:10]: nit: use double quotes for all lines

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import { loadTimelinePopup } from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const bobRossIpsum = "Just use the old one inch brush. It's so important to do something every day that will make you happy. "
+  + "I sincerely wish for you every possible joy life could bring. Water's like me. It's laaazy ... Boy, it "
+  + "always looks for the easiest way to do things\n\nWe're trying to teach you a technique here and how to use it. "
+  + 'The least little bit can do so much. There we go.\n\nNo worries. No cares. Just float and wait for the wind to blow you around. '
+  + "These trees are so much fun. I get started on them and I have a hard time stopping. We'll make some happy little bushes here. "
+  + 'Absolutely no pressure. You are just a whisper floating across a mountain. There are no limits in this world. '
+  + 'A thin paint will stick to a thick paint.\n';
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1.js:10]: same nit: use double quotes for all lines

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset1.js (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset1.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/231/commits/be1e593d65b150312127d5698a853d9f70ec4979).

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1.js (hunk 2 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/231/commits/be1e593d65b150312127d5698a853d9f70ec4979).

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/Timeline/Reflow/DynamicData.jsx [hunk 3]
```diff
@@ -5,16 +5,17 @@ import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
 import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
 import '@cerner/terra-graphs/lib/components/Graph.module.scss';
 import '@cerner/terra-graphs/lib/components/Timeline/Timeline.module.scss';
-import data1 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset';
-import data2 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset';
+import dataset1 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset1';
+import dataset2 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/defaultDataset2';
+import dynamicDataset1 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset1';
+import dynamicDataset2 from '@cerner/terra-graphs-docs/src/example-datasets/dataObjects/Timeline/criticalDataset2';
 import getTimelinePanningConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Timeline/timelinePanningConfig';
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/Timeline/Reflow/DynamicData.jsx:12]: nit: move this line above dataset imports

**Review by sdadn — APPROVED** [review]

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/Timeline/Reflow/DynamicData.jsx (hunk 3 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/Timeline/Reflow/DynamicData.jsx]: Fixed [here](https://github.com/cerner/terra-graphs/pull/231/commits/d4e147b8450f28a94d8a56f8f82e177493b6d5b6).

**Pull request merged**